hail/src/main/scala/is/hail/backend/service/Worker.scala

```diff
@@ -160,6 +160,8 @@ object Worker {
         writeString(dos, shortMessage)
         writeString(dos, expandedMessage)
         dos.writeInt(errorId)
+        log.info(s"job $i/$n failed with user exception: $shortMessage (error id $errorId)\n  $expandedMessage")
+        throw userError
```

Comment:
Nice catch here. I was wondering for a bit why worker jobs weren't failing on error.